documentation/docs/examples/cad-highlighting.mdx

```diff
@@ -106,7 +106,7 @@ viewer.on('click', event => {
         if (taggedEquipment) {
           // Highlight tagged equipment and all children
           model.selectNodeByTreeIndex(taggedEquipment.treeIndex, true);
-        }
+        } 
```

Comment:
```suggestion
        }
```

Will create a task on miro and try to handle linter for all packages, not only the viewer...